inc/links.php

```diff
@@ -132,3 +132,40 @@ function set_headless_rest_preview_link( WP_REST_Response $response, WP_Post $po
 }
 add_filter( 'rest_prepare_page', __NAMESPACE__ . '\set_headless_rest_preview_link', 10, 2 );
 add_filter( 'rest_prepare_post', __NAMESPACE__ . '\set_headless_rest_preview_link', 10, 2 );
+
+/**
+ * Redirects non-API requests for public URLs to the specified front-end URL.
+ *
+ * @see https://github.com/wpengine/faustjs/blob/canary/plugins/wpe-headless/includes/deny-public-access/callbacks.php
+ *
+ * @param object $query The current query.
+ *
+ * @return void
```

Comment:
Remove blank lines between tags and add `@author WebDevStudios` and `@since NEXT` below `@see`.

Remove `@return` (we're not doing `@return void` elsewhere).

Change `@see` link to `https://github.com/wpengine/faustjs/blob/aaad74cd6edac536a1df405552256ca66575c8cd/plugins/wpe-headless/includes/deny-public-access/callbacks.php#L20` so it's a permalink, in case they completely change their plugin structure later.